### DISCUSSION TITLE
Disable expand feature in navigation

### DISCUSSION THREAD

**Discussion by MFlisar** [squidfunk/mkdocs-material#7656] [Q&A]
Is it possible to disable the expanding feature in the navigation? I simple want a tree in the nav where I do not see any expand/collapse icons but the tree only.

![grafik](https://github.com/user-attachments/assets/cf6cb092-8948-4680-821f-a0d4a58437a1)

**Comment by alexvoss** [comment]
I assume you are aware of the [navigation sections](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-sections) and [navigation expansion](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-expansion) settings? Both together would almost take you there.

If, on top of expanding navigation, you also want to get rid of the icons for expanding/collapsing subsections then think you might need to hide these with some [extra css](https://squidfunk.github.io/mkdocs-material/customization/?h=extra+css#additional-css). Something like this should do the trick:

```css
.md-nav__icon.md-icon {
  display: none;
}
```

**Reply by MFlisar** [reply]
I'm aware of those settings. But those do not disable the expand/collapse function and also result in a short "flashing" when the navigation is expanded...
